app/gateway/changelog.md

```diff
@@ -769,6 +769,8 @@ openid-connect
   recommended to upgrade yet if you are an OpenTracing user. This will be
   rectified in an upcoming patch/minor release.
 
+* RFC3987 validation on route paths was removed, allowing operators to create a route with an invalid path URI like `/something|`. This validation will be added back in a future release.
```

Comment:
> operators

What is the correct way to refer to someone who configures Kong (I originally wrote `kong user` but that doesn't quite seem right).

Reply:
We usually say simply "users". Operators works too though.